conf/dritte.conf.part

```diff
@@ -36,6 +36,43 @@ source src_ch_mobility_standorte : def_searchable_features
         from mobility.standorte_tooltip
 }
 
+source src_ch_sem_sachplan_asyl_facilities_anhoerung : def_searchable_features
+{
+    sql_db = dritte_dev
+    sql_query = \
+        SELECT bgdi_id as id \
+            , stabil_id as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(facname_de, ' ')||' '||coalesce(facname_fr, ' '))||' '||coalesce(facname_it, ' ') as  detail \
```

Comment:
you can use `remove_accent(concat_ws(' ', facname_de, facname_fr, facname_it))` instead